apps/nextjs/src/app/_components/header.tsx

```diff
@@ -0,0 +1,55 @@
+import Link from "next/link";
+
+import { auth, signIn, signOut } from "@acme/auth";
+import { Button } from "@acme/ui/button";
+
+const sharedClasses = {
```

Comment:
FYI: I think this is a tailwind antipattern. 
I'm unsure if lsp and formatting will handle this correctly

Reply:
The shared classes? I agree 100%, I think this was me experimenting with Co-pilot and wanted it to improve my code but it rather made an anti-pattern, can be changed

Reply:
Fixed, close if you are satisfied